### DISCUSSION TITLE
[Kavita] Kindle Style Progress Indicator While Reading

### DISCUSSION THREAD

**Discussion by DieselTech** [Kareadita/Kavita#2567] [Ideas]
### Idea Description

Posted by risho
· Jun 2022
· 11 Upvotes
https://feats.kavitareader.com/posts/158/kindle-style-progress-indicator-while-reading

Description

It would be nice to have a simple absolute percentage at the bottom of the page to be able to easily track progress while you are reading. I've attached a picture to show what it looks like on the kindle

### Duration of Using Kavita

42

### Idea Category

Feature Enhancement

### Agreement

- [X] I agree that this is solely for submitting ideas, and I will search for existing ideas before posting.

**Comment by majora2007** [comment]
I believe we now have this with the new EPUB reader refresh:
<img width="501" height="102" alt="image" src="https://github.com/user-attachments/assets/bd1920a9-94db-4de9-a5b5-f0fdcaf18a12" />

Since this was ported, I will close for now. If one of the upvoters know if this doesn't, please add context.
